<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signature</class>
 <widget class="QWidget" name="Signature">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: italic 11pt &quot;Serif&quot;;
color:black;
 background: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 blue, stop:1 cyan);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>394</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
color:black;</string>
   </property>
   <property name="text">
    <string>SAVE &amp; EXIT</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>791</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="l_GetId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>GET ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_GetStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>Get Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_GetId_Ans">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_GetStatus_Ans">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_TestConnect">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
font: italic 16pt &quot;Serif&quot;;</string>
    </property>
    <property name="text">
     <string>FINGER SCANING PROCESS</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>791</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="l_GetFinger1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>Get Finger (1)</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_getimage1_Ans">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
font: italic 16pt &quot;Serif&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_gener_ans">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_GetFinger2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>Get Finger (2)</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_GetFinger3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 11pt &quot;Serif&quot;;
background:transparent;</string>
    </property>
    <property name="text">
     <string>Get Finger (3)</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_getimage2_ans">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
font: italic 16pt &quot;Serif&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_gener2_ans">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>100</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_getimage3_ans">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
font: italic 16pt &quot;Serif&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_gener3_ans">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>190</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="lPic_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>761</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lPic_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>761</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lPic_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>761</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="l_store">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>450</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_Merge">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>390</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
font: italic 14pt &quot;Serif&quot;;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_Result">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>430</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Serif&quot;;
background:transparent;
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>pushButton</zorder>
  <zorder>l_store</zorder>
  <zorder>l_Merge</zorder>
  <zorder>l_Result</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
